spv/backend.go

```diff
@@ -114,7 +144,6 @@ func (s *Syncer) LoadTxFilter(ctx context.Context, reload bool, addrs []dcrutil.
 	}
 	for i := range outpoints {
 		s.rescanFilter.AddUnspentOutPoint(&outpoints[i])
```

Comment:
hmm, i don't think this will be needed anymore either

Reply:
`spv/Syncer` still uses the filter's stored outpoint to determine whether a particular transaction is relevant so for the moment we can't remove this call from here.

Specifically: https://github.com/decred/dcrwallet/blob/master/spv/rescan.go#L45

Reply:
ok, we'll leave it now then. it's something we should look into removing though (just pass the transactions to the wallet and let it open a db view to determine if it spends a relevant output, instead of keeping each of them on the heap).